indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -536,6 +539,11 @@ def can_withdraw(self, user, check_state=False):
     def can_see_reviews(self, user):
         return self.can_judge(user) or self.can_convene(user)
 
+    def get_timeline(self, user=None):
+        comments = [each for each in self.comments if each.can_view(user)] if user else self.comments
```

Comment:
Yeah I think it was even me showing it in some other project's code finding it funny :P